RELEASE.md

```diff
@@ -0,0 +1,138 @@
+## Automated Release
```

Comment:
@hairyhum please remove all the Kasten K10 related information from this file. Maybe don't add this file until we move the entire release process to GH.

Reply:
Moved to https://github.com/kanisterio/kanister/pull/2829